README.md

```diff
@@ -128,13 +128,16 @@ where,
 Demo is deployed from the application code in `demo/` folder.
 Link: https://tarkalabs.github.io/tarka-chat/demo/
 
-## Dev notes
-
-If you wanted to run this code in your local for development purposes, Run the following command from project root folder `yarn dev` and open `dev/index.html` in your browser to see the chat window. Your local changes will be hot reloaded
-
-## Old versions
+## All versions
 
 - v1.1 : https://d1fmfone96g0x2.cloudfront.net/tarka-chat-1.1.umd.js
```

Comment:
The versions available in the CDN so far are - `1.0`, `2.0.1`, `2.0.2`, `2.1.0`, `2.2.0`, `2.2.1`, `2.3.0`, `2.3.1` and `2.4.0`